patch/kconfig-inclusions

```diff
@@ -126,6 +126,182 @@ CONFIG_SENSORS_EMC2305=m
+CONFIG_CRYPTO_NULL=y
+CONFIG_CRYPTO_RNG=y
+CONFIG_CRYPTO_RNG_DEFAULT=y
+CONFIG_CRYPTO_SEQIV=y
+CONFIG_CRYPTO_SHA1_ARM64_CE=y
+CONFIG_CRYPTO_SHA256_ARM64=y
+CONFIG_CRYPTO_SHA2_ARM64_CE=y
+CONFIG_CRYPTO_SHA512_ARM64=y
+CONFIG_CRYPTO_SIMD=y
+CONFIG_DEFAULT_SECURITY_DAC=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_DM_BUFIO=y
+CONFIG_DM_VERITY=y
+CONFIG_DW_WATCHDOG=y
+CONFIG_EEPROM_AT24=y
+CONFIG_EEPROM_AT25=y
+CONFIG_EMBEDDED=y
+CONFIG_EXT3_FS=y
+CONFIG_EXT4_FS=y
+CONFIG_FAT_FS=y
+CONFIG_FS_MBCACHE=y
+CONFIG_GENERIC_IRQ_DEBUGFS=y
+CONFIG_GPIO_DWAPB=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_DESIGNWARE_CORE=y
+CONFIG_I2C_DESIGNWARE_PLATFORM=y
+CONFIG_I2C_RD1173=y
+CONFIG_INET_AH=y
+CONFIG_INET_DIAG=y
+CONFIG_INET_ESP=y
+CONFIG_INET_IPCOMP=y
+CONFIG_INET_TCP_DIAG=y
+CONFIG_INET_TUNNEL=y
+CONFIG_INET_XFRM_TUNNEL=y
+CONFIG_IPV6_SIT=y
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_MANGLE=y
+CONFIG_IP_NF_MATCH_AH=y
+CONFIG_IP_NF_MATCH_ECN=y
+CONFIG_IP_NF_MATCH_TTL=y
+CONFIG_IP_NF_TARGET_ECN=y
+CONFIG_IP_NF_TARGET_REJECT=y
+CONFIG_IP_NF_TARGET_TTL=y
+CONFIG_IP_PNP=y
+CONFIG_IP_PNP_BOOTP=y
+CONFIG_IP_PNP_DHCP=y
+CONFIG_IRQSOFF_TRACER=y
+CONFIG_JBD2=y
+CONFIG_KALLSYMS_ALL=y
+CONFIG_LEGACY_PTYS=y
+CONFIG_LIBCRC32C=y
+CONFIG_LOCALVERSION_AUTO=y
+CONFIG_MDIO_DEVICE=y
+CONFIG_MEMTEST=y
+CONFIG_MMC_DEBUG=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_CADENCE=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MQ_IOSCHED_KYBER=y
+CONFIG_MTD=y
+CONFIG_MTD_OF_PARTS=y
+CONFIG_MTD_SPI_NOR=y
+CONFIG_NETFILTER_XTABLES=y
+CONFIG_NETFILTER_XT_MATCH_DSCP=y
+CONFIG_NETFILTER_XT_MATCH_ECN=y
+CONFIG_NETFILTER_XT_MATCH_HL=y
+CONFIG_NETFILTER_XT_MATCH_PKTTYPE=y
+CONFIG_NETFILTER_XT_MATCH_POLICY=y
+CONFIG_NETFILTER_XT_TARGET_DSCP=y
+CONFIG_NETFILTER_XT_TARGET_HL=y
+CONFIG_NET_IP_TUNNEL=y
+CONFIG_NET_KEY=y
+CONFIG_NET_PKTGEN=y
+CONFIG_NF_REJECT_IPV4=y
+CONFIG_NLS_CODEPAGE_437=y
+CONFIG_NLS_ISO8859_1=y
+CONFIG_NO_HZ_FULL=y
+CONFIG_OVERLAY_FS=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_PMBUS=y
+CONFIG_PPS=y
+CONFIG_PREEMPTIRQ_TRACEPOINTS=y
+CONFIG_PTP_1588_CLOCK=y
+CONFIG_RCU_NOCB_CPU=y
+CONFIG_RING_BUFFER_ALLOW_SWAP=y
+CONFIG_RTC_DRV_PCF85363=y
+CONFIG_SCHED_TRACER=y
+CONFIG_SENSORS_LM90=y
+CONFIG_SENSORS_LTC2978=y
+CONFIG_SENSORS_PMBUS=y
+CONFIG_SENSORS_TPS53679=y
+CONFIG_SERIAL_8250_DEPRECATED_OPTIONS=y
+CONFIG_SOFT_WATCHDOG=y
+CONFIG_SPI_CADENCE_QUADSPI=y
+CONFIG_SPI_DESIGNWARE=y
+CONFIG_SPI_DW_MMIO=y
+CONFIG_SQUASHFS=y
+CONFIG_SQUASHFS_DECOMP_MULTI=y
+CONFIG_SQUASHFS_FILE_CACHE=y
+CONFIG_SYSFS_SYSCALL=y
+CONFIG_TRACER_SNAPSHOT_PER_CPU_SWAP=y
+CONFIG_TRACE_IRQFLAGS=y
+CONFIG_TUN=y
+CONFIG_VFAT_FS=y
+CONFIG_VIRT_CPU_ACCOUNTING=y
+CONFIG_VIRT_CPU_ACCOUNTING_GEN=y
+CONFIG_VLAN_8021Q=y
+CONFIG_XFRM_AH=y
+CONFIG_XFRM_ALGO=y
+CONFIG_XFRM_ESP=y
+CONFIG_XFRM_IPCOMP=y
+CONFIG_XFRM_USER=y
+CONFIG_XOR_BLOCKS=y
+CONFIG_XZ_DEC_ARM=y
+CONFIG_XZ_DEC_ARMTHUMB=y
+CONFIG_XZ_DEC_BCJ=y
+CONFIG_XZ_DEC_IA64=y
+CONFIG_XZ_DEC_POWERPC=y
+CONFIG_XZ_DEC_SPARC=y
+CONFIG_XZ_DEC_X86=y
```

Comment:
A lot of these configs are set by default, or are not related to the platform at all. Can you reduce this to the platform-specific configs?

Reply:
I compared the configs which are present with't pensando platform specific changes which is in src/sonic-linux-kernel/linux-5.10.140/debian/build/build_arm64_none_arm64/.config and added whichever config is missing in the new patch.